implementation/src/main/java/io/smallrye/openapi/api/util/ArchiveUtil.java

```diff
@@ -221,6 +221,9 @@ private static boolean acceptClassForScanning(OpenApiConfig config, String archi
         }
         String fqcn = archivePath.replaceAll("/", ".").substring(0, archivePath.lastIndexOf(OpenApiConstants.CLASS_SUFFIX));
         String packageName = "";
+        if (fqcn.startsWith(".")) {
+            fqcn = fqcn.substring(1);
+        }
```

Comment:
This will work, but I wonder if it would be cleaner to do a substring(1) up on line 158 and then remove the leading `/` from `OpenApiConstants.WEB_ARCHIVE_CLASS_PREFIX`. Thoughts?